src/app/modules/form/form.service.ts

```diff
@@ -343,3 +344,59 @@ export const retrievePublicFormsWithSmsVerification = (
     return okAsync(forms)
   })
 }
+
+export const createSampleSubmissionData = (
+  sampleData: Record<string, unknown>,
+  field: FormFieldDto,
+) => {
+  let sampleValue = null
+  let noOfOptions = 0
+  let randomSelectedOption = 0
+  switch (field.fieldType) {
+    case 'textarea':
+    case 'textfield':
+      sampleValue =
+        'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.'
+      break
+    case 'radiobutton':
+    case 'dropdown':
+      noOfOptions = field.fieldOptions.length
+      randomSelectedOption = Math.floor(Math.random() * noOfOptions)
+      sampleValue = field.fieldOptions[randomSelectedOption]
+      break
+    case 'email':
+      sampleValue = 'hello@example.com'
+      break
+    case 'decimal':
+      sampleValue = 1.234
+      break
+    case 'number':
+      sampleValue = 1234
+      break
+    case 'mobile':
+      sampleValue = '+6598765432'
+      break
+    case 'homeno':
+      sampleValue = '+6567890123'
+      break
+    case 'yes_no':
+      sampleValue = 'yes'
+      break
+    case 'rating':
+      sampleValue = 1
+      break
+    case 'attachment':
+      sampleValue = 'attachmentFileName'
+      break
+    default:
+      break
+  }
+  if (sampleValue != null) {
+    sampleData[field._id] = {
+      question: field.title,
```

Comment:
Isn't the field id supposed to be part of the object as well in responses?

Reply:
Added in [3f2dcc1](https://github.com/opengovsg/FormSG/pull/6325/commits/3f2dcc19ac6a66253f05f172c83cb3dd2d70a458)